### PR TITLE
Fix webdriver test

### PR DESCRIPTION
**Pull request by maggie-lou**
For prod and dev targets, the builds may have cached data from previous runs

### CODE REVIEW

**Comment by maggie-lou** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3331** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/3331" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/3331?utm_source=stack-comment).

**Review by siggisim — APPROVED** [review]
I'll approve if this is blocking, but I'll let @bduffany decide if we want to test this caching stuff in a different way (or maybe we should do something like `--noremote_accept_cached` on the first build in dev/prod.

**Comment by bduffany** [comment]
`--noremote_accept_cached` seems reasonable; we could also change the test to use a unique `--remote_instance_name` (e.g. generate it as a UUID at the start of the test) and then inspect only AC results in the scorecard.

**Comment by maggie-lou** [comment]
> I'll approve if this is blocking, but I'll let @bduffany decide if we want to test this caching stuff in a different way (or maybe we should do something like `--noremote_accept_cached` on the first build in dev/prod.

Good idea - added

**Review by siggisim — APPROVED** [review]

**Review by bduffany — APPROVED** [review]

**Pull request merged**
